contracts/DocumentStore.sol

```diff
@@ -1,103 +1,80 @@
 pragma solidity ^0.5.10;
```

Comment:
fixing the solc version helps with reproducing the bytecode for verification - previously this was done in truffle-config but can do it in the pragma too

Reply:
done